profile/README.md

```diff
@@ -34,3 +34,11 @@ This requirement sets a bar for transparency, and a foundation for future innova
 
 This section will be developed as the project gets under way. In the mean time, feel free to reach out directly to team members or through our [GitHub
 Discussions](https://github.com/orgs/ccmmf/discussions).
+
+## Licenses and Copyright
+
+A central focus of this project is to generate open source software and open data that will enable transparency while facilitating reuse, collaboration, and derivative works. To this end, we plan to release our works under the following licenses:
+   
+- Software: BSD 3-Clause or compatible license
+- Writing: [CC-BY](https://creativecommons.org/licenses/by/4.0/)
+- Data Products: Public domain ([CC0](https://creativecommons.org/public-domain/cc0/)).
```

Comment:
```suggestion
- Data Products: Public domain ([CC0](https://creativecommons.org/public-domain/cc0/)).

If any work in a CCMMF repo is missing its LICENSE file or its terms of use are otherwise not clear, please tell us -- that is a bug and we will fix it.
```

Reply:
I don't think this is necessary, since any repository missing a LICENSE file will inherit https://github.com/ccmmf/.github/blob/main/LICENSE

Reply:
TBH this is more to be reassuring to the reader than for us -- "we just showed you a list, but there will not be a quiz. It's not your job to figure out whether we thought a given file was 'writing' or 'data' for licensing purposes."